_ext/eclipse-base/README.md

```diff
@@ -0,0 +1,66 @@
+# spotless-eclipse-base
+
+Eclipse formatters are part of the the Eclipse User Interface implementations. Hence their public interfaces are depending on various Eclipse modules.
```

Comment:
Am I right to understand from this paragraph that, the way Eclipse works internally, formatters depend on details from the user interface part(s) of Eclipse itself?

If yes, then perhaps this paragraph would read better as something like this?

> Eclipse formatters are intrinsically linked to the user interface of Eclipse proper. As a consequence, the formatters' public interfaces depend on various, seemingly-unrelated Eclipse modules.

Reply:
I am afraid the GUI is not the problem. I've rewritten it as follows:
> Eclipse formatters are embedded in plugins serving multiple purposes and not necessarily supporting headless builds. Hence the plugin interfaces are depending on various Eclipse plugins and services not required for the formatting purpose.

Reply:
LGTM. Ta. :)